bot/exts/info/doc/_cog.py

```diff
@@ -342,29 +340,15 @@ async def get_command(self, ctx: commands.Context, *, symbol_name: Optional[str]
 
             if doc_embed is None:
                 error_message = await send_denial(ctx, "No documentation found for the requested symbol.")
+                await wait_for_deletion(error_message, (ctx.author.id,), timeout=NOT_FOUND_DELETE_DELAY)
+                with suppress(discord.NotFound):
+                    await error_message.clear_reaction(Emojis.trashcan)
 
-                if ctx.message.mentions or ctx.message.role_mentions:
-                    await error_message.add_reaction(DELETE_ERROR_MESSAGE_REACTION)
-
-                    try:
-                        await self.bot.wait_for(
-                            'reaction_add',
-                            check=lambda reaction, user: reaction.message == error_message and user == ctx.author and str(reaction) == DELETE_ERROR_MESSAGE_REACTION,
-                            timeout=NOT_FOUND_DELETE_DELAY
-                        )
-
-                        with suppress(discord.HTTPException):
-                            await error_message.delete()
-
-                    except asyncio.TimeoutError:
-                        await error_message.clear_reaction(DELETE_ERROR_MESSAGE_REACTION)
-
-                else:
-                    await wait_for_deletion(error_message, (ctx.author.id,), timeout=NOT_FOUND_DELETE_DELAY)
+                # Make sure that we won't cause a ghost-ping by deleting the message
+                if not (ctx.message.mentions or ctx.message.role_mentions):
                     with suppress(discord.NotFound):
                         await ctx.message.delete()
-                    with suppress(discord.NotFound):
-                        await error_message.delete()
```

Comment:
Is this change intentional? Prior, both the invoking message and the error embed would be deleted, but now the error embed remains. I noticed that this is a duplicate commit of the other ghost ping PR, but the situation is still similar to the pypi command.

Reply:
Yeah, I changed this deliberately assuming we'd want consistent behaviour with the docs command. If you think this should be reverted to the old behaviour then I'll do so, but my thoughts was that this should be updated to be the same as docs.

Reply:
For now I'll leave this conversation unsolved and wait for your response @kosayoda.

Reply:
I am asking this question for both this and the docs PR. `wait_for_deletion` does not delete the embed after the timeout. Before the docs PR, the error embed would be deleted after the timeout (code being the removed lines), which is no longer the case now.

Reply:
Right, I see. I personally prefer keeping the embed and just removing the reactions, but if we want to revert to old system we can do.

Reply:
For these embeds specifically, the idea is to not have multiple error embeds lying around, since they have lose meaning right after being shown. This is in line with the short configured delay on the reaction check.

Reply:
I think we should add the embed delete inside the same `suppress()`, that way the embed only gets deleted if the bot is able to delete the invoking message.

See this diff:
```diff
                # Make sure that we won't cause a ghost-ping by deleting the message
                if not (ctx.message.mentions or ctx.message.role_mentions):
                    with suppress(discord.NotFound):
                        await ctx.message.delete()
+                       await error_message.delete()
```

If the invoking message pings someone, and they did not react with a trashcan, then both messages will stay up.
If the invoking messaes does not ping someone both messages will be deleted regardless of reacting with a trashcan in time.

Reply:
That sounds good to me. I'll do this if @kosayoda thinks that's good too.

Reply:
Kosayoda has approved on discord so will commit this :+1: